system/integrations/pgp/index.rst

```diff
@@ -0,0 +1,111 @@
+PGP
+======
+
+
+Introduction
+------------
+
+PGP is another method for secure email communication.
```

Comment:
Let's define PGP acronym before we use it for the first time (i.e. `Pretty Good Privacy (PGP) is...`).